config/common.js

```diff
@@ -88,5 +89,11 @@ module.exports = function(env){
         'components': path.resolve(__dirname,'../src/components')
       }
     },
+    plugins: [
```

Comment:
Put this back -- the webpack JSON loader was making the file much larger bc of the way it was processing it. Loading it via an xhr.